### PR TITLE
task/3956-review-rspec

### PR DESCRIPTION
**Pull request by buitiendo**
redmine: https://edu-redmine.framgia.vn/issues/3956

### CODE REVIEW

**Comment by ghost** [comment]
ok

**Comment by buitiendo** [comment]
ready

File: spec/controllers/reviews_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RSpec.describe Admin::ReviewsController, type: :controller do
+  let!(:user) {FactoryBot.create(:user)}
+  let!(:review) {FactoryBot.create :review}
+  
+  describe "GET index" do
+    before :each do
+      session[:user_id] = user.id
+    end
+
+    it "assigns @reviews" do
```
Comment (huongnguyenmta) [inline spec/controllers/reviews_controller_spec.rb:13]: viết test case invalid nữa nhé c ơi

**Comment by buitiendo** [comment]
ready

**Comment by buitiendo** [comment]
review_model
rspec_controller

**Comment by buitiendo** [comment]
ready

**Pull request merged**